### PR TITLE
Added changelog to 1.5.0 RC1

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

**Comment by mrsolo** [comment]
add to whitelist

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,55 @@
+1.5.0.rc1
+  # general
+  - You can now use LS_HOME/patterns directory to add generic patterns for those that may not be
+    associated with a particular plugin. Patterns in this dir will be loaded by default (#2225)
+  - We now check if the config file is correctly encoded. Otherwise we show a verbose error message
+    to convert the failing config file(s) to UTF-8 (#LOGSTASH-1103)
+  - Fixed bug in pipeline to gracefully teardown output workers when num workers > 1
```
Comment (jordansissel) [inline CHANGELOG]: d6cb066e038d53fd2bdb47c0c05487b929aab801 fixed this, ticket #2180

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+  - Fixed nologin path in release debian packages (#2283)
+  - Resolved issue where Logstash was crashing for users still using exclude_tags in their output
+    configuration (#2323)
+  - Updated and releases a new version of the logstash-event gem with latest changes in 1.5.0 RC1
+  - Windows: Significantly improved the initial user experience with Windows platform. Fixed many
+    issues in File input. Added support for using the plugin framework (installing, upgrading, removing)
+  - With the release of Kibana 4, we have removed the `bin/logstash web` command and any reference to
+    Kibana from Logstash. Kibana can be started very simply by running `bin/kibana` from the installed directory
```
Comment (jordansissel) [inline CHANGELOG]: > Kibana can be started very simply by running `bin/kibana` from the installed directory
I think this sentence can be removed.

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+
+  # input
+  - Added new CouchDB input plugin to fetch data from CouchDB. Using the _changes API, data can be kept
```
Comment (jordansissel) [inline CHANGELOG]: Might be explicit and follow with "This is intended to be a replacement for the Elasticsearch Couch River" or whatever

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+    in sync with any output like Elasticsearch by using this input
+  - File Input: Fixed a number of issues on Windows platform. These include resolving file locking
+    issues which was causing log files to not rotate. Also added support for using SinceDB to record
+    multiple files' last read information. Also fixed encoding issues which applies to many inputs (#2507). 
+  - Kafka: Added support for whitelisting and blacklisting topics in the input. 
+  - EventLog: For Windows, this input gracefully shutsdown if there is a timeout while receiving events
+  - Heartbeat: We created a new input plugin for generating heartbeat messages at periodic intervals. 
+    Use this to monitor Logstash -- you can measure the latency of the pipeline using these dummy events,
```
Comment (jordansissel) [inline CHANGELOG]: > these dummy events,

"these heartbeat events"

**Comment by suyograo** [comment]
@jordansissel Updated

File: CHANGELOG [hunk 5]
```diff
@@ -1,3 +1,55 @@
...
+    to capture failed requests from Elasticsearch and retry them. Error codes like 429 (too many requests) will
+    now be retried by default for 3 times. The number of retries and the interval between consecutive retries 
+    can be configured
+  - S3: Fixed a critical problem in the S3 Output plugin when using the size_file option. This could cause data loss
+    and data corruption of old logs. Added IAM roles support so you can securely read and write events from S3 
+    without providing your AWS credentials. Added support for using temporary credentials obtained from AWS STS
+
+  # codec
+  - Added new Elasticsearch bulk codec which can be used to read data formatted in the Elasticsearch 
+    Bulk API (multiline json) format. For example, this codec can be used in combination with RabbitMQ input
```
Comment (untergeek) [inline CHANGELOG]: If anything can be added here, I'd say something about its functionality being like the Elasticsearch RabbitMQ river

**Comment by untergeek** [comment]
lgtm

File: CHANGELOG [hunk 6]
```diff
@@ -1,3 +1,72 @@
...
+    to convert the failing config file(s) to UTF-8 (#LOGSTASH-1103)
+  - Fixed bug in pipeline to gracefully teardown output workers when num workers > 1 (#2180)
+  - Fixed nologin path in release debian packages (#2283)
+  - Resolved issue where Logstash was crashing for users still using exclude_tags in their output
+    configuration (#2323)
+  - Updated and releases a new version of the logstash-event gem with latest changes in 1.5.0 RC1
+  - Windows: Significantly improved the initial user experience with Windows platform (#2504, #1426). 
+    Fixed many issues related to File input. Added support for using the plugin 
+    framework (installing, upgrading, removing)
+  - With the release of Kibana 4, we have removed the `bin/logstash web` command and any reference to
```
Comment (jordansissel) [inline CHANGELOG]: 945e184e67b9515f2ac5d5c1dea3d7c2b72b55af, #2661

File: CHANGELOG [hunk 7]
```diff
@@ -1,3 +1,72 @@
...
+      AWS credentials (#1575). 
+    - Added support for using temporary credentials obtained from AWS STS (#1946)
+  - Lumberjack: Resolved issue where unrelated events were getting merged into a single event while using
+    this input with with the multiline codec (#2016)
+
+  # filter
+  - Mutate: 
+    - Resolved issue where convert option was creating an extra field in the event (#2268)
+    - Fixed issue where mutate with non-existent field was throwing an error (#2379)
+  - Multiline: Handled cases where we dropped lines from certain multiline xml messages (#3)
```
Comment (jordansissel) [inline CHANGELOG]: maybe "where we unintentionally deduplicated lines, such as repeated lines in xml messages"

**Comment by jordansissel** [comment]
Other than my current comments, LGTM.

I suspect, moving forward, doing changelog will be *much* easier as there won't really be any plugin changes to note, just saying "Plugin X is version Y", right? :)

**Comment by suyograo** [comment]
Should have squashed before merge, my bad :(

**Pull request closed without merging**